### PR TITLE
Geminus City Tweaks/ Theta Project Change

### PR DESCRIPTION
**Pull request by lngredients**
- Changed name of eyeballs to Augmented Eyeballs in theta room.
- Changed name of Enhanced Heart to Augmented Heart in theta room.
- Expanded the Theta area to accommodate a new closet.
- Added in a Project Theta Closet.
- Added in Theta Project Armor, Helmet, Arm Guards, boots and undersuit. (All unique items with custom descriptions- All not as good as Spartan armor)
- Added in Augmented Lungs. Unique item comparable to Augmented Liver.
- Changed Max damage for Augmented Eyes to 60 from 45 to be slightly tougher as it doesn't offer that much of a benefit other then flash resistance.
- Adding in three stacks of ten sheets of glass into cargo storage area.
- Adding in two stacks of ten sheets of glass into the Engineering Area of Geminus.
- Added a Sink to the Padded Cell in Hospital.
- Added additional sink closer to Surgery table at GCPD Morgue.
- Moved Microwave in hospital break room so people can access the Getmore Vendor.
- Added in recharger at hospital where spare Defibs are kept.
- Added in sink to hospital Morgue.
- Added holopads inside a number of buildings outside of the slums. (10-15 extra Mostly in government/public buildings)
- Added sink to Viro Bathroom.

### CODE REVIEW

**Review by DoctorOctopusTheOctopi — DISMISSED** [review]
Hiya! I'll be peer-reviewing your stuff today. There is just a minor issue that can be resolved with a quick tutorial on github, but you're currently trying to push all of your changes into the main repository under a new area called geminus_city

https://gyazo.com/e3d56ea18a38fb85c24bf3fa2d13ad99

instead of this area right here

https://gyazo.com/28c657bc7347201123cba679e731fb90

If you need help rectifying this issue I'll be in discord my ckey is Octopus#9682

Thank you for your interest in developing with us boys at Halostation and we hope you stick around and stuff, because we need more devboys.

**Comment by lngredients** [comment]
Alright I'll try to change it. Thanks.

**Comment by bloxgate** [comment]
Hi, I noticed that the PR description mentions several code changes, such as augmented lungs, however the changes list only shows the map changes.

Are you sure you committed everything?

**Comment by lngredients** [comment]
So what I did was create a lung object, then directly edited the variables of that single object. Same with the armor. So I didn't need to create any new code, I just modified existing objects to essentially create new ones.

**Comment by bloxgate** [comment]
So, what then, does the augmented lung do? The Augmented Liver actually does have some bonuses to it, hence the "comparable to" makes me expect the lungs to do stuff as well.

**Comment by lngredients** [comment]
Yeah, it's tougher then your average lung, and I was thinking about changing breath-ability and toxin resistance, but I am not familiar enough with how the numbers contained in those variables would effect gameplay to do so. So they are, for now, just tougher lungs.

**Comment by bloxgate** [comment]
I'm not sure if those values should be changed through just var-edits. If they're just tougher for now, fair enough then.

**Review by bloxgate — CHANGES_REQUESTED** [review]
Map contains at least one occurrence of step_x or step_y. These should either be removed or changed to pixel_x/pixel_y as needed.

**Comment by lngredients** [comment]
It's essentially the same as changing armor values on the armors I made. When I tested it it all seemed to work.

**Comment by lngredients** [comment]
"Map contains at least one occurrence of step_x or step_y. These should either be removed or changed to pixel_x/pixel_y as needed." Not sure what this means.

**Comment by bloxgate** [comment]
I was referring to editing their toxin resistance and such, just changing the health should be fine AFAIK.

**Comment by lngredients** [comment]
Yeah, like I said, I have no reference to what those would do, so I left those values alone.

**Comment by bloxgate** [comment]
When editing the map, you made something appear inside a wall by adjusting its step_x or step_y variables (or hitting the nudge button). These are used for pixel movement, and break movement in game if used in a map.

You should change the apparent location using pixel_x or pixel_y instead.

**Review by bloxgate — DISMISSED** [review]

**Comment by bloxgate** [comment]
~~Leave this marked DNM for now. @lngredients mentioned some problems in Discord that need to be sorted first.~~

File: baystation12.dme [hunk 1]
```diff
@@ -2617,10 +2617,11 @@
 #include "maps\crashsite_zeta\crashsite_zeta_map.dm"
 #include "maps\desert_outpost\__desert_outpost_map.dm"
 #include "maps\example\example_define.dm"
-#include "maps\first_contact\all_includes.dm"
-#include "maps\first_contact\mapdef.dm"
```
Comment (DoctorOctopusTheOctopi) [inline baystation12.dme]: Why'd you uncheck these two files? If you don't mind me asking

File: baystation12.dme [hunk 2]
```diff
@@ -2617,10 +2617,11 @@
 #include "maps\crashsite_zeta\crashsite_zeta_map.dm"
 #include "maps\desert_outpost\__desert_outpost_map.dm"
 #include "maps\example\example_define.dm"
-#include "maps\first_contact\all_includes.dm"
-#include "maps\first_contact\mapdef.dm"
+#include "maps\geminus_city\geminus_city_jobs.dm"
 #include "maps\geminus_city\geminus_city_map.dm"
+#include "maps\geminus_city\geminus_city_outfits.dm"
 #include "maps\geminus_city\geminus_city_overmap.dm"
+#include "maps\geminus_city\geminus_city_spawns.dm"
```
Comment (DoctorOctopusTheOctopi) [inline baystation12.dme]: These includes don't need to be in here, the city/jobs, city/map, city/outfits, and city/spawn.. Uncheck those.. Travis is screaming at it.

[UNIQUENESS: Outfit Datums Shall Have Unique Names]: Names - Marine - The following entries have the same value: /decl/hierarchy/outfit/job/marine and /decl/hierarchy/outfit/job/UNSC_ship/marine
## UNIT_TEST ##: !!! FAILURE !!! [UNIQUENESS: Outfit Datums Shall Have Unique Names]: 1 issue with outfit datums found.
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Colonist - Missing HUD icon: hudcolonist
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Colonist - Insurrectionist Sympathiser - Missing HUD icon: hudcolonistinsurrectionistsympathiser
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Colonist - Insurrectionist Recruiter - Missing HUD icon: hudcolonistinsurrectionistrecruiter
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Mayor - Missing HUD icon: hudmayor
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Marine - Missing HUD icon: hudmarine
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Marine - Squad Leader - Missing HUD icon: hudmarinesquadleader
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: GCPD Officer - Missing HUD icon: hudgcpdofficer
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Chief of Police - Missing HUD icon: hudchiefofpolice
## UNIT_TEST ##: !!! FAILURE !!! [JOB: Shall have a HUD icon]: icons/mob/hud.dmi - 0 failed sanity checks, 8 jobs with missing HUD icon.

**Review by DoctorOctopusTheOctopi — DISMISSED** [review]

File: baystation12.dme (hunk 1 above)
Comment (lngredients) [reply baystation12.dme]: They told me to uncheck all the first contact files

File: baystation12.dme (hunk 2 above)
Comment (lngredients) [reply baystation12.dme]: Hmm... let me check and see

Comment (lngredients) [reply baystation12.dme]: It might have been from the new push I made last night

File: baystation12.dme (hunk 1 above)
Comment (lngredients) [reply baystation12.dme]: Should I recheck them?

File: baystation12.dme (hunk 2 above)
Comment (lngredients) [reply baystation12.dme]: Oh, no, I was told that the overmap and spawns were needed. Shall I uncheck those as well?

Comment (DoctorOctopusTheOctopi) [reply baystation12.dme]: I'd just unchange whatever you changed the latest commit. It was fine the second to last commit, but the last commit threw Travis into a loop. Do you know what you did?

Comment (lngredients) [reply baystation12.dme]: All I did was copy and paste the map file from my original cloned files (the one I worked on before and submmited via the website) into my repository branch on my local github. The map file is the same one from the very first commit, so I didn't think there should be any issue. How do I undo the last push?

Comment (lngredients) [reply baystation12.dme]: That should do it, I reverted the "Branch Fix commit"

Comment (lngredients) [reply baystation12.dme]: Try running the check now.

**Comment by bloxgate** [comment]
```
git revert 28ee34a9433500c7ac8802f7a37a7f5bb92a3e53
```

File: baystation12.dme (hunk 2 above)
Comment (bloxgate) [reply baystation12.dme]: You need to push the revert commit

Comment (lngredients) [reply baystation12.dme]: Right, should be pushed now.

Comment (lngredients) [reply baystation12.dme]: I accidently reverted the map fix that Blox did, so I re-reverted

Comment (lngredients) [reply baystation12.dme]: FML

Comment (lngredients) [reply baystation12.dme]: I'll pray to the code gods that that did it. :(

Comment (lngredients) [reply baystation12.dme]: Oh hold on I need to uncheck the files I think.

**Review by DoctorOctopusTheOctopi — APPROVED** [review]
Looks golden. I can't see anything wrong with the raw .dmm or any items/objects that worry me. Ready for a live test. Author said they playtested. 👍 🐙

**Pull request merged**